addon/services/raven.js

```diff
@@ -4,7 +4,7 @@ export default RavenService.extend({
     releaseMetaName: 'sentry:revision',
     release: Ember.computed({
         get: function() {
-            return $("meta[name='revision']").attr('content');
+            return $("meta[name='sentry:revision']").attr('content');
```

Comment:
Updated it to use releaseMetaName, although it doesn't look like any other metaName is supported https://github.com/dschmidt/ember-cli-deploy-sentry/blob/6f7a610d9e9b6f95ededbdbec0f4ede1e0b1dce7/index.js#L69
Is that a future feature?